torch/csrc/jit/script/module.h

```diff
@@ -197,18 +235,21 @@ struct TORCH_API Module {
     AT_ERROR("Method '", name, "' is not defined.");
   }
 
-  Module get_module(const std::string& name) const {
-    auto obj = module_object()->getAttr(name).toObject();
-    return Module(obj);
-  }
+  void apply(const std::function<void(Module&)>& fn);
 
-  ivalue_list get_slots() const;
+  buffer_list buffers(bool recurse = true) const;
+  named_buffer_list named_buffers(bool recurse = true) const;
 
-  module_list get_modules() const;
+  module_list children() const; // direct modules
+  named_module_list named_children() const;
```

Comment:
We should add a comment explaining the difference between `named_children` and `named_modules`. Ideally, add a comment for all these methods.